examples/quickstart/jupyter-notebooks/nested-columns-tutorial.ipynb

```diff
@@ -0,0 +1,556 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "240b0ad5-48f2-4737-b12b-5fd5f98da300",
+   "metadata": {},
+   "source": [
+    "## Transform nested data\n",
+    "\n",
+    "You can use Druid's [SQL JSON functions](https://druid.apache.org/docs/latest/querying/sql-json-functions.html) to transform nested data in your ingestion query.\n",
+    "\n",
+    "Run the following cell to insert sample data into a new datasource named `kttm_transform`. The SELECT query extracts the `country` and `city` elements from the nested `geo_ip` column and creates a composite object `sessionDetails` containing  `session` and `session_length`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1f21fa8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql/task\"\n",
+    "print(druid_host+endpoint)\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "\"query\": \"INSERT INTO \\\"kttm_transform\\\" \\\n",
+    "    WITH \\\"source\\\" AS \\\n",
+    "    (SELECT * FROM TABLE(EXTERN('{\\\"type\\\":\\\"http\\\",\\\"uris\\\":[\\\"https://static.imply.io/example-data/kttm-nested-v2/kttm-nested-v2-2019-08-25.json.gz\\\"]}', \\\n",
+    "       '{\\\"type\\\":\\\"json\\\"}','[{\\\"name\\\":\\\"timestamp\\\",\\\"type\\\":\\\"string\\\"},{\\\"name\\\":\\\"session\\\",\\\"type\\\":\\\"string\\\"},{\\\"name\\\":\\\"session_length\\\",\\\"type\\\":\\\"string\\\"}, \\\n",
+    "        {\\\"name\\\":\\\"event\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"},{\\\"name\\\":\\\"agent\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"},{\\\"name\\\":\\\"geo_ip\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"}]'))) \\\n",
+    "        SELECT TIME_PARSE(\\\"timestamp\\\") AS \\\"__time\\\", \\\n",
+    "        JSON_QUERY(geo_ip, '$.country') as country, \\\n",
+    "        JSON_QUERY(geo_ip, '$.city') as city, \\\n",
+    "        JSON_OBJECT('session':session, 'session_length':session_length) as sessionDetails \\\n",
+    "        FROM \\\"source\\\" \\\n",
+    "    PARTITIONED BY DAY\"\n",
+    "})\n",
+    "\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "ingestion_taskId_response = response\n",
+    "ingestion_taskId = json.loads(ingestion_taskId_response.text)['taskId']\n",
+    "\n",
+    "print(f\"\\nInserting data into the table named {dataSource2}\")\n",
+    "print(\"\\nThe response includes the task ID and the status: \" + response.text + \".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a72f7a9-d7f3-4987-bce5-cb482369b1c3",
+   "metadata": {},
+   "source": [
+    "Run the following cell to get the status of the ingestion task."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd275148-b930-4626-a433-e34a1c308f03",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
```

Comment:
You don't need to import time a second time. The cell that also had the sleep loop already imported it.

Reply:
Updated